### DISCUSSION TITLE
How would I go about getting this working with C# mono

### DISCUSSION THREAD

**Discussion by Pa1yn-dev** [utopia-rise/fmod-gdextension#147] [Q&A]
If someone could give me pointers it'd be much appreciated!

**Comment by CedNaru** [comment]
This a C++ plugin. You can't directly use it from C#, only GDScript can.
Indirectly, you could use it from C# the same way you would call GDScript from C#, which is very cumbersome to do:
[Calling GDScript from C#](https://docs.godotengine.org/en/stable/tutorials/scripting/cross_language_scripting.html#calling-gdscript-methods-from-c)

**Reply by Pa1yn-dev** [reply]
I mean to utilise unity's integration which uses C#, instead to use godot. Worst case scenario calling GDScript from C# could work.

**Reply by CedNaru** [reply]
I don't understand. The unity plugin for FMOD won't work in Godot obviously. And if you are asking about FMOD itself, we are using the C++ api, not the C# one.
